db/migrate/20210503194732_create_events.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEvents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.text :description
+      t.string :location
+      t.datetime :date
+      t.integer :creator_id
```

Comment:
- [ ] [required] Events table in migrations has creator_id as the foreign key ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from Odin)